### DISCUSSION TITLE
Then newest version isn't available as Ubuntu package

### DISCUSSION THREAD

**Discussion by okadsistemas** [tjko/jpegoptim#95] [General]
The available version on Ubuntu is 1.4.3. The lastest version is 1.4.6

Ref: https://packages.ubuntu.com/source/xenial/jpegoptim

**Comment by tssajo** [comment]
Jah, in old versions of Ubuntu.
In newer versions of Ubuntu (such as Disco, Eoan, Focal) you'll find the newer version.
All you need to do is: upgrade your old Ubuntu system!! :)
